src/common/dlfcn_win32.cc

```diff
@@ -15,6 +15,7 @@
 #include <sstream>
 #include <windows.h>
 
+#include "common/errno.h"
```

Comment:
Actually, there's another issue. win32_errno.h ends up being used by files that expect C linkage (e.g. librados.h): http://paste.openstack.org/raw/793113/

Checks added to win32_errno.h wouldn't help because of ``extern "C"`` wrappers from a higher level.